src/main/webapp/components/CheckboxGroupsComponent.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import ColumnLayout from '@splunk/react-ui/ColumnLayout';
+import Text from '@splunk/react-ui/Text';
+import { isFalse } from '../util/util';
+import { StyledColumnLayout, StyledSwitch } from './StyledComponent';
+
+function CheckboxGroupsComponent(props) {
+    const { field, label, value, checkboxTextFieldValue, handleChange } = props;
+
+    const [isDisabled, setIsDisabled] = useState(!value);
+
+    useEffect(() => {
+        setIsDisabled(!value);
+    }, [value]);
+
+    const handleChangeCheckbox = () => {
+        if (value && !isFalse(value)) {
```

Comment:
why didn't you use isTrue() ?

Reply:
Nice catch!! Let me use isTrue and will make sure that everything is working fine.